bff-node/src/util/InitializaitonUtils.ts

```diff
@@ -3,11 +3,13 @@ import '../config/env';
 import { DefaultAzureCredential } from '@azure/identity';
 import { AppConfigurationClient } from '@azure/app-configuration';
 import * as appInsights from 'applicationinsights';
-import { bffVersion } from '..';
+import { bffVersion, isLocal } from '..';
+import { resolve } from 'path';
```

Comment:
I don't think this method is being used as you believe or had intended. Same reasons as the previous comment on the resolve. 😊